### PR TITLE
Update wasm features support in Node.js

### PR DESCRIPTION
**Pull request by chicoxyzzy**
Node.js 16.4 ships with V8 9.1

https://github.com/v8/v8/blob/master/src/wasm/wasm-feature-flags.h

### CODE REVIEW

File: features.json [hunk 1]
```diff
@@ -153,17 +153,19 @@
 		"Node.js": {
 			"url": "https://nodejs.org/",
 			"logo": "/images/nodejs.svg",
-			"version": "15.0",
+			"version": "16.4",
 			"features": {
 				"bigInt": true,
 				"bulkMemory": true,
+				"exceptions": "--experimental-wasm-eh"
 				"multiValue": true,
```
Comment (RReverser) [inline features.json:161]: Github rightfully highlights a syntax error here :) (missing comma on previous line)

File: features.json [hunk 2]
```diff
@@ -153,17 +153,19 @@
...
 				"mutableGlobals": true,
-				"referenceTypes": "--experimental-wasm-anyref",
+				"referenceTypes": "--experimental-wasm-reftypes",
 				"saturatedFloatToInt": true,
 				"signExtensions": true,
-				"simd": "--experimental-wasm-simd",
-				"threads": "--experimental-wasm-threads"
+				"simd": true,
+				"tailCall": "--experimental-wasm-return-call"
```
Comment (RReverser) [inline features.json:167]: Same here.

File: features.json [hunk 3]
```diff
@@ -153,17 +153,19 @@
...
-			"version": "15.0",
+			"version": "16.4",
 			"features": {
 				"bigInt": true,
 				"bulkMemory": true,
+				"exceptions": "--experimental-wasm-eh",
 				"multiValue": true,
 				"mutableGlobals": true,
-				"referenceTypes": "--experimental-wasm-anyref",
+				"referenceTypes": "--experimental-wasm-reftypes",
```
Comment (chicoxyzzy) [inline features.json:163]: it was renamed year ago https://github.com/v8/v8/commit/ba688c6ec9c0fb4439f7381cb8046ea14b159d0e#diff-73af4fa91a5bf27a89d47cce1271322131f71f54223cea49939723a72f5d3f6e

**Comment by RReverser** [comment]
Thanks!

**Pull request merged**